UKHomeOffice/cerberus-service
# COP-11890 - Vehicle details (field highlighting)

## Description
This PR adds field level highlighting to the vehicle component.

## To Test
- Pull & run against dev
- Post a task to dev
- Post an update to the same task excluding (risks & selectors) e.g. for an Accompanied movement, modify the vehicle details and keep track of the changed fields.
- The task should now have an update, within the latest version, the changed fields should also be highlighted.
